src/components/BounceRulesContainer/RuleListContainer/index.scss

```diff
@@ -1,4 +1,4 @@
-table {
+.rule-table {
```

Comment:
nice thanks for changing some of these class names to be less CSS specific and more aligned with a class name